Algorithms/Sorting/Tree_Sort.py

```diff
@@ -0,0 +1,84 @@
+''' SORTING USING BINARY SEARCH TREE
+    Binary Search Tree is a special type of binary tree where
+    1. The value of all the nodes in the left sub-tree is less than or equal to the value of the root.
```

Comment:
Binary tree can't have duplicate elements, so the left subtree can't have equal value of the root node

Reply:
Here Tree is used to sort a list of numbers . The list may contain duplicate elements which needs to be stored in the tree